content/docs/studio/user-guide/single-sign-on.md

```diff
@@ -0,0 +1,98 @@
+AD, etc.
+
+SSO for teams can be configured by [team `admins`](#roles), and requires
+configuration on both DVC Studio and the IDP. The exact steps for this depend on
+the IDP.
+
+Once the SSO configuration is complete users can login to DVC Studio by opening
+their team's login page `http://studio.iterative.ai/api/teams/<TEAM_NAME>/sso`
+in their browser. They can also login directly from their Okta end-user
+dashboards by clicking on the DVC Studio integration icon.
+
+If a user does not have a pre-assigned role when they sign in to a team, they
+will be auto-assigned the [`Viewer` role](#roles).
+
+## Okta integration
+
+1. **Create Enterprise account**: SSO is available for DVC Studio teams with
+   enterprise account subscriptions. If you are on the Free or Basic plan of DVC
+   Studio, [Contact us] to upgrade your account.
+
+2. **Add integration with DVC Studio in Okta**: Follow the instructions from the
+   [Okta developer guide](https://developer.okta.com/docs/guides/build-sso-integration/saml2/main/#create-your-integration-in-okta).
+   In short, login to Okta with an admin account, and follow these steps:
+
+   1. In the admin console, go to `Applications` -> `Create App Integration` to
+      create a private SSO integration.
+   2. Use `SAML 2.0` as the `Sign in method` (and not `OIDC` or some other
+      option).
+   3. Enter any name (eg, `DVC Studio`) as the `App name`.
+   4. `Single sign-on URL`:
+      [https://studio.iterative.ai/api/teams/<TEAM_NAME>/saml/consume](https://studio.iterative.ai/api/teams/<TEAM_NAME>/saml/consume)
+      (Replace <TEAM_NAME> with the name of your team in Studio.
+   5. `Audience URI (SP Entity ID)`: https://studio.iterative.ai/api/saml
+   6. `Name ID Format`: Persistent
+   7. `Application username (NameID)`: Okta username
+   8. `Attribute Statements (optional)`: 1. `Name`: email 2. `Name format`: URI
+      Reference 3. `Value`: user.email
+
+   ![](https://static.iterative.ai/img/studio/sso_okta_configure_saml.png)
+
+   8. Click on `Next` and `Finish`.
+   9. Once the integration is created, open the `Sign On` tab and expand the
+      `Hide Details` section. From here, copy the `Sign on URL`, `Issuer` and
+      `Signing Certificate`.
+
+   ![](https://static.iterative.ai/img/studio/sso_okta_sign_on_details.png)
+
+3. **Configure Okta in DVC Studio team settings**: Back in Studio, open your
+   enterprise team's `Settings` page. Go to the `SAML Single Sign-on` section
+   and enable SAML Authentication.
+
+   ![](https://static.iterative.ai/img/studio/sso_enable_saml_authentication.png)
+
+   Then, fill in the required details:
+
+   1. `Sign-on URL`: Paste the `Sign on URL` you copied from Okta
+   2. `Identity Provider Issuer URL`: Paste the `Issuer` you copied from Okta
+   3. `Public Certificate`: Paste the `Signing Certificate` you copied from
+      Okta. Remember to add `-----BEGIN CERTIFICATE-----` and
```

Comment:
> Remember to add `-----BEGIN CERTIFICATE-----` ..

are those included when you copy it? can we make it more flexible? do we have a check on FE for this?

Reply:
we do have FE and BE checks. These are not available when you copy the certificate from Okta.
I had the same experience while configuring GitHub SAML

We could probably leave only the backend validation. Check like the certificate, in case of a validation error try to prepend and append the "BEGIN/END CERTIFICATE" and check again.

Reply:
I have added this issue to change the validation process https://github.com/iterative/studio/issues/9145